core/runtime/tests/error.rs

```diff
@@ -115,3 +115,53 @@ fn syntax_error() {
   let frame = js_error.frames.first().unwrap();
   assert_eq!(frame.column_number, Some(12));
 }
+
+#[tokio::test]
+async fn aggregate_error() {
+  let mut runtime = JsRuntime::new(Default::default());
+  let src = r#"
+(async () => {
+  await Promise.any([]);
+})()
+"#;
+  let value_global = runtime
```

Comment:
Any chance these could be integration and/or unit tests? I'd like to avoid adding more of these tests which are a bit awkward to maintain in Rust.